includes/ucf-social-common.php

```diff
@@ -102,19 +116,164 @@ public static function has_social_feed( $content ) {
 			// [ucf-social-feed] shortcode, as well as processed string
 			// contents for substrings that are likely to be present in social
 			// feed templates
-			if (
-				has_shortcode( $content, 'ucf-social-feed' )
-				|| ( strpos( $content_processed, 'new Curator.' ) !== false || strpos( $content_processed, 'ucf-social-feed' ) !== false )
-			) {
+			if ( has_shortcode( $content, 'ucf-social-feed' ) || strpos( $content_processed, 'ucf-social-feed' ) !== false ) {
 				$has_feed = true;
 			}
 
 			return $has_feed;
 		}
 
+		/**
+		 * Returns a JSON object from the provided URL.  Detects undesirable status
+		 * codes and returns false if the response doesn't look valid.
+		 *
+		 * @since 3.0.0
+		 * @author Jo Dickson
+		 * @param string $url URL that points to a JSON object/feed
+		 * @return mixed JSON-decoded object or false on failure
+		 */
+		private static function fetch_json( $url ) {
+			$response      = wp_remote_get( $url, array( 'timeout' => 15 ) );
```

Comment:
If there's no reason to have this specifically at 15, it would be nice to have this configurable or just shorter.